bookomate/settings/base.py

```diff
@@ -20,10 +22,10 @@
 # See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '3_kr)66f*&-x2a_(f_!9%4lqbe^@eiuj4+ez(&9*wy*klyvp5t'
+SECRET_KEY = config('DJANGO_KEY')
```

Comment:
I think `config('SECRET_KEY')` instead `DJANGO_KEY` will be better.

Reply:
I will change it.

Question: with this change, will travis ci pass?

Reply:
With aditional env variable (postgresql) in .travis.yml should pass.

Reply:
But, you do that part, right?

Reply:
@mateuszz0000 I commit the requested changes.